extensions/sql-database-projects/package.json

```diff
@@ -455,8 +455,11 @@
     "vscode-languageclient": "^5.3.0-next.1",
     "vscode-nls": "^4.1.2",
     "which": "^2.0.2",
-    "xml-formatter": "^2.1.0",
-    "xmldom": "^0.6.0"
+    "xml-formatter": "2.1.0",
```

Comment:
xml-formatter was getting updated to 2.6.1 which caused building errors. couldn't figure out the fix so changed it to a fix version for now